guides/client/apollo.md

```diff
@@ -171,18 +165,20 @@ phoenixSocket.conn.close();
 
 ## Using both HTTP and websocket links
 
-A common configuration for Apollo client applications is to use both HTTP and websocket links -- HTTP for queries and mutations, and a websocket for subscriptions. We can implement this in our client using [directional composition with Apollo's `split` helper](https://www.apollographql.com/docs/link/composition#directional).
+A common configuration for Apollo client applications is to use both HTTP and websocket links -- HTTP for queries and mutations, and a websocket for subscriptions. We can implement this in our client using [directional composition with Apollo's `split` helper](https://www.apollographql.com/docs/react/api/link/introduction/#directional-composition).
 
 ```javascript
-import ApolloClient from "apollo-client";
-import { createHttpLink } from "apollo-link-http";
-import { setContext } from "apollo-link-context";
+import {
+  ApolloClient,
+  InMemoryCache,
+  createHttpLink,
+  split
+} from "@apollo/client";
+import { getMainDefinition } from "@apollo/client/utilities";
+import { setContext } from "@apollo/client/link/context";
 import * as AbsintheSocket from "@absinthe/socket";
 import { createAbsintheSocketLink } from "@absinthe/socket-apollo-link";
 import { Socket as PhoenixSocket } from "phoenix";
-import { hasSubscription } from "@jumpn/utils-graphql";
```

Comment:
To avoid an additional dependency in the user's codebase, I've gone ahead and updated this example to use our recommendation in the [directional composition](https://www.apollographql.com/docs/react/api/link/introduction/#directional-composition) section of our docs.